File: src/Services/WareHouse/WareHouse.API/Application/Model/WareHouseLimitDTO.cs
    using System;
using System.Collections.Generic;

#nullable disable

namespace WareHouse.API.Application.Model
{
    public partial class WareHouseLimitDTO:BaseModel
    {
        public string WareHouseId { get; set; }
        public string ItemId { get; set; }
        public string UnitId { get; set; }
        public string UnitName { get; set; }
        public int MinQuantity { get; set; }
        public int MaxQuantity { get; set; }
        public DateTime CreatedDate { get; set; }
        public string CreatedBy { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string ModifiedBy { get; set; }
        public string WareHouseName{get;set;}
        public string ItemName{get;set;}
        public virtual IEnumerable<WareHouseItemDTO> WareHouseItemDTO { get; set; }
        public virtual IEnumerable<UnitDTO> UnitDTO { get; set; }
        public virtual IEnumerable<WareHouseDTO> WareHouseDTO { get; set; }
        public virtual WareHouseItemDTO Item { get; set; }
        public virtual UnitDTO Unit { get; set; }
        public virtual WareHouseDTO WareHouse { get; set; }
    }
}

File: src/WebClient/WareHouse/src/app/model/WareHouseLimitDTO.ts
import { BaseModel } from "./BaseModel";
import { WareHouseItemDTO } from 'src/app/model/WareHouseItemDTO';
import { UnitDTO } from 'src/app/model/UnitDTO';
import { WareHouseDTO } from 'src/app/model/WareHouseDTO';

export interface WareHouseLimitDTO extends BaseModel {
    wareHouseId: string|null;
    itemId: string|null;
    unitId: string|null;
    unitName: string|null;
    minQuantity: number;
    maxQuantity: number;
    createdDate: string|null;
    createdBy: string|null;
    modifiedDate: string|null;
    modifiedBy: string|null;
    item: WareHouseItemDTO|null;
    unit: UnitDTO|null;
    wareHouse: WareHouseDTO|null;
    wareHouseItemDTO: WareHouseItemDTO[]|null;
    unitDTO: UnitDTO[]|null;
    wareHouseDTO: WareHouseItemDTO[]|null;
}